seatunnel-e2e/seatunnel-connector-v2-e2e/connector-kafka-e2e/src/test/resources/kafkasink_fake_to_kafka.conf

```diff
@@ -0,0 +1,68 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+env {
+    execution.parallelism = 1
+    job.mode = "BATCH"
+
+    #spark config
+    spark.app.name = "SeaTunnel"
+    spark.executor.instances = 1
+    spark.executor.cores = 1
+    spark.executor.memory = "1g"
+    spark.master = local
+}
+
+source {
+  FakeSource {
+    row.num = 10
+    map.size = 10
+    array.size = 10
+    bytes.length = 10
+    string.length = 10
+    schema = {
+      fields {
+        c_map = "map<string, smallint>"
+        c_array = "array<int>"
+        c_string = string
+        c_boolean = boolean
+        c_tinyint = tinyint
+        c_smallint = smallint
+        c_int = int
+        c_bigint = bigint
+        c_float = float
+        c_double = double
+        c_decimal = "decimal(30, 8)"
+        c_null = "null"
+        c_bytes = bytes
+        c_date = date
+        c_timestamp = timestamp
+      }
+    }
+  }
+}
+
+transform {
+}
+
+sink {
+  Kafka {
+    bootstrap.servers = "kafkaCluster:9092"
+    topic = "test_topic"
+    partition_key = ["c_map","c_string"]
```

Comment:
How to use a fixed string partition key value?  eg: partition_key = 'aaa'

Reply:
It is recommended to implement in this way, which is simpler for users to use

partition_key = "${c_map},${c_string}"

Reply:
The value of partition_key can be a normal string or an expression to extract the field content. We have already supported single field extraction before

https://github.com/apache/incubator-seatunnel/pull/3085

Reply:
@hailin0 You want to keep the logic of following code snippet, right?
`
        if (!fieldNames.contains(partitionKey)) {
            return row -> partitionKey;
        }
`

Reply:
yes

Reply:
> 
BTW, The fixed string partition key value will cause to write all of data to a partition of topic, specify or omit partition key(s) is good practice.

Reply:
I agree

I'm worried about breaking the released feature

Reply:
Perhaps rename config key to `partition_key_fields` is more appropriate

Reply:
and need to update the examples in the docs

<img width="1427" alt="image" src="https://user-images.githubusercontent.com/14371345/199939588-9530dcd7-ba6b-4c4c-b626-42591f96d255.png">

Reply:
done.